lib/Graph.dart

```diff
@@ -109,21 +120,25 @@ class Graph {
     return _nodes[position];
   }
 
-  Node getNodeAtUsingData(Widget data) => _nodes.firstWhere((element) => element.data == data);
+  Node getNodeAtUsingData(Widget data) =>
+      _nodes.firstWhere((element) => element.data == data);
```

Comment:
💡 this should return a nullable `Node`, cos if I will pass a widget that is not a part of node - it will throw `StateError`. So we also need to use `firstWhereOrNull` here

Reply:
The same for methods below